src/components/card/card.scss

```diff
@@ -77,6 +40,43 @@
             .col-sm-9 {
                 position: relative;
             }
+
+            .validation-message {
+                $arrow-border-width: 1rem;
+                display: block;
+                position: absolute;
+                top: 0;
+                left: 0;
+                transform: translate(16rem, 0);
+                margin-left: $arrow-border-width;
+                border: 1px solid $active-gray;
+                border-radius: 5px;
+                background-color: $ui-orange;
+                padding: 1rem;
+                max-width: 18.75rem;
+                min-height: 1rem;
+                overflow: visible;
+                color: $type-white;
+
+                &:before {
+                    display: block;
+                    position: absolute;
+                    top: 1rem;
+                    left: -$arrow-border-width / 2;
+
+                    transform: rotate(45deg);
+
+                    border-bottom: 1px solid $active-gray;
+                    border-left: 1px solid $active-gray;
+                    border-radius: 5px;
+
+                    background-color: $ui-orange;
+                    width: $arrow-border-width;
+                    height: $arrow-border-width;
+
+                    content: "";
+                }
+            }
```

Comment:
`.form` was already there – I was mostly trying to get this under `.row`, since I don't think we necessarily want it to be styled the same way outside of `.row`. I'm not sure if we'd want to move `.row` outside of `.form` though. Or just move `.validation-message` outside again, working under the pretense that we most likely would never have use for it in another way on `.card`.